src/org/opendatakit/common/web/constants/BasicConsts.java

```diff
@@ -54,6 +54,7 @@
   public static final String RIGHT_BRACKET = "]";
   public static final String LEFT_PARENTHESIS = "(";
   public static final String RIGHT_PARENTHESIS = ")";
+  public static final String INVALID_CHARS = "[^a-zA-Z0-9\\ \\-\\_]";
```

Comment:
@yanokwa,

I added the _ and - and I'm using explicit space character instead of \s now.